server/plugin.go

```diff
@@ -331,13 +332,15 @@ func (p *Plugin) GetMeetingOAuth(meetingID int, userID string) (*zoom.Meeting, e
 	}
 
 	client := conf.Client(ctx, zoomUserInfo.OAuthToken)
+	client.Timeout = 10 * time.Second
```

Comment:
Nit: Using `client.Timeout` sets a timeout of that specific client, not for the full method (which makes little to no difference) here. Using `context.WithTimeout` is the more idiomatic way, as it's models a timeout for the full method.